indicator/controller.py

```diff
@@ -854,7 +854,7 @@ def create_indicator_action_classification_and_practice(action, scope, title, da
     csv_file = io.StringIO()
     writer = csv.writer(csv_file)
     for data_ in data.values():
-        writer.writedata(data_['data'])
+        writer.writerows(data_['data'])
```

Comment:
@rafaelpezzuto dúvida o `data_['data']` pode ser dict ou tem que ser um json?

Reply:
@robertatakenaka Segue descrição oficial do que o writerows espera:

https://docs.python.org/3/library/csv.html: "Write all elements in rows (an iterable of row objects as described above) to the writer’s file object, formatted according to the current dialect."

Como é um arquivo CSV, espera-se que cada linha seja aquele carinha separado por vírgulas. Quando trabalho com CSV, costumo usar o csv.DictWriter informando as colunas que o arquivo terá. E o data usando em "writerows" costuma ser uma lista de dicionários.

Então, por exemplo, seria possível fazer:

```python
data = [
    {
        "nome": "Rafael", "sobrenome": "Damaceno", "id": 1,
        "nome": "Fernanda "sobrenome": "Rodrigues", "id": 2,
    }
]
with open("meuarquivo.csv", "w") as fout:
    escritor = csv.DictWriter(fout, delimiter=",", fieldnames=["id", "nome", "sobrenome"])
    escritor.writerows(data)
```
de modo a gerar um arquivo CSV no seguinte formato:

```
id,nome,sobrenome
1,rafael,damaceno
2,fernanda,rodrigues
```